src/openms/source/ANALYSIS/OPENSWATH/SpectrumAddition.cpp

```diff
@@ -41,8 +41,46 @@
 namespace OpenMS
 {
 
-  OpenSwath::SpectrumPtr SpectrumAddition::addUpSpectra(const std::vector<OpenSwath::SpectrumPtr>& all_spectra,
-      double sampling_rate, bool filter_zeros)
+  void sortSpectrumByMZ(OpenSwath::Spectrum& spec)
```

Comment:
this method would be nice to have a member of the Spectrum?
I can see this being useful in a number of scenarios and people will probably not find this implementation.
btw: this kind of functionality already exists for MSSpectrum, so this code duplication is a bit unfortunate, but I can see why this was done.